UIKitDz/ViewController.swift

```diff
@@ -7,6 +7,162 @@ import UIKit
 class ViewController: UIViewController {
```

Comment:
final - ставим перед всеми классами для производительности и ускорения горячего билда.
```suggestion
final class ViewController: UIViewController {
```